ksqldb-engine/src/test/java/io/confluent/ksql/schema/ksql/inference/DefaultSchemaInjectorTest.java

```diff
@@ -299,7 +328,8 @@ public void shouldInjectForCsStatement() {
             + "`mapField` MAP<STRING, BIGINT>, "
             + "`structField` STRUCT<`s0` BIGINT>, "
             + "`decimalField` DECIMAL(4, 2)) "
-            + "WITH (KAFKA_TOPIC='some-topic', KEY_FORMAT='protobuf', KEY_SCHEMA_ID=18, VALUE_FORMAT='avro', VALUE_SCHEMA_ID=5);"
+            + "WITH (KAFKA_TOPIC='some-topic', KEY_AVRO_SCHEMA_FULL_NAME='myrecord', "
+            + "KEY_FORMAT='protobuf', VALUE_AVRO_SCHEMA_FULL_NAME='myrecord', VALUE_FORMAT='avro');"
```

Comment:
Nor sure why `KEY_AVRO_SCHEMA_FULL_NAME` and `KEY_AVRO_SCHEMA_FULL_NAME` are added here. The test setup seems to use `givenKeyAndValueInferenceSupported` that does not specify a explicit schema-ids ?